crates/topos-tce-api/src/grpc/builder.rs

```diff
@@ -63,7 +63,6 @@ impl ServerBuilder {
         }));
 
         let console = ConsoleServiceServer::new(TceConsoleService {
-            command_sender: command_sender.clone(),
```

Comment:
maybe we can keep this as we may add more command in the future to manage a node.